### PR TITLE
Fix install extra plugin issue and Adds more translations

### PR DESCRIPTION
**Pull request by dhrupo**

### CODE REVIEW

File: app/Http/Controllers/LoggerController.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function delete(Request $request, Logger $logger)
         }
         
         return $this->sendSuccess([
-            'message' => "{$subject} deleted successfully."
+            'message' => __("{$subject} deleted successfully.", 'fluent-smtp')
```
Comment (techjewel) [inline app/Http/Controllers/LoggerController.php:46]: This should be using __(sprintf())

File: app/Http/Controllers/LoggerController.php [hunk 2]
```diff
@@ -99,14 +99,14 @@ public function retryBulk(Request $request, Logger $logger)
                 $failedInitiated++;
             }
         }
-        $message = 'Selected Emails have been proceed to send.';
+        $message = __('Selected Emails have been proceed to send.', 'fluent-smtp');
 
         if($failedCount) {
-            $message .= ' But '.$failedCount.' emails are reported to failed to send.';
+            $message .= __(' But ', 'fluent-smtp') . $failedCount . __(' emails are reported to failed to send.', 'fluent-smtp');
```
Comment (techjewel) [inline app/Http/Controllers/LoggerController.php:105]: Let's use sprintf as well

File: app/Http/Controllers/LoggerController.php [hunk 3]
```diff
@@ -99,14 +99,14 @@ public function retryBulk(Request $request, Logger $logger)
...
         }
 
         if($failedInitiated) {
-            $message .= ' And '.$failedInitiated.' emails are failed to init the emails';
+            $message .= __(' And ', 'fluent-smtp') . $failedInitiated . __(' emails are failed to init the emails', 'fluent-smtp');
```
Comment (techjewel) [inline app/Http/Controllers/LoggerController.php:109]: also sprintf

**Review by techjewel — CHANGES_REQUESTED** [review]
Please make these changes

**Comment by dhrupo** [comment]
Fixed and pushed

https://github.com/WPManageNinja/fluent-smtp/pull/192/commits/eb9d52a05db808041c78cb8d5b18e17925cf4e36

**Pull request merged**